src/ostorlab/cli/docker_requirements_checker.py

```diff
@@ -18,6 +19,24 @@ def is_docker_installed() -> bool:
             return False
     return True
 
+def is_sys_arch_supported() -> bool:
+    """Checks if the systems cpu architecture is supported
+       The architecture is checked with a return value that's based on the kernel implementation of the uname(2)
+       system call. So it might be necesarry to handle the same arch with various strings e.g. linux returns x86_64
+       or AMD64 on windows.
+    Returns:
+        True if the architecture is supported, else False
+    """
+    supported_arch_types = ["x86_64", "AMD64"]
+    if sys.platform == 'darwin' and 'ARM' in platform.version():
+        """On mac os, uname returns x86 even on arm64 if the process calling it is running via rosetta. We parse for ARM
```

Comment:
This should be a comment style, I think the linter will not pass this. As FYI, we apply Google style guide in the code base for consistency : https://google.github.io/styleguide/pyguide.html